mailheader.js

```diff
@@ -126,8 +125,8 @@ class Header {
 
     _remove_more (key) {
         const key_len = key.length;
-        for (let i=0,l=this.header_list.length; i < l; i++) {
-            if (this.header_list[i].substring(0, key_len).toLowerCase() === key) {
+        for (let i=0, l=this.header_list.length; i < l; i++) {
```

Comment:
Can't we do `const header of this.header_list` here too?

Reply:
If you notice, `i` gets used within the loop (in the splice). Kinda important...

Reply:
I should probably look into getting new glasses.